concepticondata/conceptlists/Gauchat-1925-480.tsv

```diff
@@ -0,0 +1,481 @@
+ID	NUMBER	ENGLISH	FRENCH	LATIN	Page	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Gauchat-1925-480-1	1	It is	Il fait	facit	2		
+Gauchat-1925-480-2	2	hot	chaud	caldu	2	1286	HOT
+Gauchat-1925-480-3	3	today	aujourd‘hui	hŏdie	2	1283	TODAY
+Gauchat-1925-480-4	4	This is	C’est	est	2		
+Gauchat-1925-480-5	5	a good	un bon	bŏnu	2	1035	GOOD
```

Comment:
The thing with list is that each gloss here should not be treated individually but rather in a context of surrounding words because obviously this list is just a bunch of sentences broken down into phrases. Here the sentence goes 'This is a good weather' (the noun 'weather' being apparently countable in French) so it just happened that the article is joined to the adjective.

Reply:
True, but this would mean that the elicited forms also contain all the parts in that segment. So this one also contains the article if the elicited dialect uses indefinite articles.

Reply:
And that is less of a problem with nouns, but since this one is an adjective, if this segment then stands alone, it looks as if the article would nominalize it. So if anyone would use that data in wordlist format, this would not only change the grammatical aspects of the word but turn it into a different part of speech.

Reply:
@Schweikhard, you are right in one sense, but it doesn't matter much, as the data is in such a clear form that we will be able to separate all the words, in fact, in our lexibank version, we already have them separated. But for the sake of consistency, we could leave it unmapped here and later see to which degree we can make another list of extracted concepticon concept sets, where articles etc. are deleted and we have the "raw words" with their occurrence and different forms (in fact, also something nice). In any case: given how specific this dataset is, there is no real "wrong" and "right" in this case.